Removed software "Network Tools"

StephanStS opened the pull request:
Removed `iftop`, `IPTraf`, `Iperf`, `MTR-Tiny`, `nLoad`, `tcpdump` from dietpi-software
- Very rare usage.
- Software is removed, otherwise documentation would be needed.
- See also https://github.com/MichaIng/DietPi-Docs/issues/649

Removed Software category "Network Tools"

Software `Avahi Daemon` moved to category "Advanced Networking"


Remark: Uninstall procedure from Iperf was missing (`G_AGP iperf`) in `dietpi-software`.

ToDo: 
- DietPi software list has to be updated (https://github.com/MichaIng/DietPi/wiki/DietPi-Software-list, IDs 10..15)
- Avahi Daemon needs to be described (see https://github.com/MichaIng/DietPi-Docs/issues/649#issuecomment-2572435380)

StephanStS committed:
Removed software "Network Tools"

Removed iftop, IPTraf, Iperf, MTR-Tiny, nLoad, tcpdump from dietpi-software
- Very rare usage.
- Software is removed, otherwise documentation would be needed.

Removed Software category "Network Tools"

Software Avahi moved to category "Advanced Networking"

MichaIng commented on CHANGELOG.txt:
```diff
@@ -8,6 +8,14 @@ Enhancements:
+- DietPi-Software| Iperf: Removed software package. Can be installed easily via `apt install iperf`, because there are no dependencies to other packages. Usage of this software was very rare.
+- DietPi-Software| MTR-Tiny: Removed software package. Can be installed easily via `apt install mtr`, because there are no dependencies to other packages. Usage of this software was very rare.
+- DietPi-Software| nLoad: Removed software package. Can be installed easily via `apt install nload`, because there are no dependencies to other packages. Usage of this software was very rare.
+- DietPi-Software| tcpdump: Removed software package. Can be installed easily via `apt install tcpdump`, because there are no dependencies to other packages. Usage of this software was very rare.
```
Let's combine those, and we may remove some more:
```suggestion
- DietPi-Software | A number of software options have been removed, which were installed just as single APT package, without any configuration or dependant. Running `dietpi-software` for such is overkill, as one can just use the native Debian package manager directly, like "apt install tcpdump". Affected software titles: iftop, IPTraf, Iperf, MTR-Tiny, nLoad, tcpdump
```

MichaIng committed:
v9.10

- DietPi-Patches | Remove obsolete install states
- DietPi-Survey_report | Remove network tools from tracked software titles

StephanStS committed:
Update CHANGELOG.txt

Co-authored-by: MichaIng <micha@dietpi.com>
